src/components/breadcrumb/__tests__/CdrBreadcrumb.spec.js

```diff
@@ -41,81 +41,96 @@ const itemsB2 = [
+    describe('a11y requirements', () => {
+      it('the breadcrumb is a "NAV" element', () => {
+        expect(wrapper.element.tagName).toBe('NAV');
+      });
 
-  it('breadcrumb should truncate with 3 or more items', async () => {
-    const wrapper = mount(CdrBreadcrumb, {
-      propsData: {
-        id: 'bc-test',
-        items: itemsA
-      }
-    });
+      it('the breadcrumb nav has the expected "aria-label" value', () => {
+        expect(wrapper.attributes()['aria-label']).toBe('breadcrumbs');
+      });
+
+      it('the ellipse button has the expected "aria-label" value', () => {
+        expect(ellipse.attributes()['aria-label']).toBe('show 1 more navigation level');
+      });
+
+      it('the ellipse button has the expected "aria-controls" value', () => {
+        expect(ellipse.attributes()['aria-controls']).toBe('bc-testList');
+      });
 
+      it('the ellipse button has the expected "aria-expanded" value', () => {
+        expect(ellipse.attributes()['aria-expanded']).toBe('false');
+      });
+    })
 
-    expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(true);
-    wrapper.find('.cdr-breadcrumb__ellipses').trigger('click');
-    await wrapper.vm.$nextTick();
-    expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBeFalsy();
+    describe('when truncated breadcrumb items list has been expanded', () => {
+      beforeEach(() => {
+        wrapper.find('.cdr-breadcrumb__ellipses').trigger('click');
+      });
+
+      it('is no longer truncated', () => {
+        expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBeFalsy();
+      });
+    })
   });
 
-  it('breadcrumb should evaluate truncation when items are updated', async () => {
-    const wrapper = mount(CdrBreadcrumb, {
-      propsData: {
-        id: 'bc-test',
-        items: itemsB
-      }
+  describe('when there is fewer than 3 items', () => {
+    let wrapper;
+    beforeEach(() => {
+      wrapper = mount(CdrBreadcrumb, {
+        propsData: {
+          id: 'bc-test',
+          items: itemsB,
+        }
+      });
     });
-    expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(false);
-    await wrapper.setProps({items: itemsB.concat(itemsB2)});
-    await wrapper.vm.$nextTick();
-    expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(true);
-  });
+    it('is not truncated', () => {
+      expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(false);
+    });
+
+    it('breadcrumb should evaluate truncation when items are updated', async () => {
+      expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(false);
+      await wrapper.setProps({ items: itemsB.concat(itemsB2) });
+      await wrapper.vm.$nextTick();
+      expect(wrapper.find('.cdr-breadcrumb__ellipses').exists()).toBe(true);
+    });
+  })
+});
+
+
+//The active element is <body> after a click. This may be an issue with the test environment. We may need document.activeElement tests to be e2e tests --Kenji
```

Comment:
Can't seem to test change of focus in this environment. All tests for document.activeElement are commented out

Reply:
Maybe try this? https://vue-test-utils.vuejs.org/api/options.html#attachto

Based on this answer here => https://stackoverflow.com/questions/65044900/why-does-document-activeelement-not-point-to-the-focused-element-in-jsdom-when-t

Reply:
Cool I'll check it out

Reply:
Yup it worked